skopi/detector/simple_square.py

```diff
@@ -28,7 +28,7 @@ def __init__(self, N_pixel, det_size, det_distance, beam=None):
         sep = float(det_size) / N_pixel
         x = -det_size/2 + sep/2 + ar*sep
         y = -det_size/2 + sep/2 + ar*sep
-        X, Y = xp.meshgrid(x, y, indexing='xy')
+        Y, X = xp.meshgrid(x, y, indexing='xy')
         Xar, Yar = xp.meshgrid(ar, ar, indexing='xy')
```

Comment:
Feels inconsistent to have Y,X in line 31 and Xar,Yar in line 32.
I suppose changing to Yar,Xar will give wrong results?

Reply:
Changing Xar,Yar to Yar,Xar, the result becomes the following (the same as keeping the original):
![image](https://user-images.githubusercontent.com/23045561/112402186-48a4b100-8cc9-11eb-99b0-638e29db0664.png)